homework03/slices/slices2.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "fmt"
+
+func main() {
+
+	a := []int64{1, 2, 5, 15}
+	fmt.Println(reverse(a))
+}
+
+func reverse(a []int64) []int64 {
+	reverseArray := make([]int64, len(a), cap(a))
+	reverseArray = a[3:4]
```

Comment:
What are these constants? You should create a function which reverses slices of different lengths. Please redo this